cookieconsent.php

```diff
@@ -58,12 +58,30 @@ public function onTwigTemplatePaths()
     public function onTwigSiteVariables()
     {
         $twig = $this->grav['twig'];
+        $assets = $this->grav['assets'];
+        $cookieConsentOptions = $this->config['plugins.cookieconsent'];
 
-        $config = $this->config->toArray();
+        // Configuration
+        $builtInCss = $cookieConsentOptions['built_in_css'];
+        $builtInJs = $cookieConsentOptions['built_in_js'];
+        $loadJsOptionsBottom = $cookieConsentOptions['load_js_options_bottom'];
 
-        $this->grav['assets']->addCss("//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.3/cookieconsent.min.css");
-        $this->grav['assets']->addJs("//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.3/cookieconsent.min.js");
+        // Load cookie consent CSS if enabled
+        if (!$this->isAdmin() && $builtInCss) {
 
-        $this->grav['assets']->addInlineJs($twig->twig->render('partials/cookieconsent.html.twig', array('config' => $config)));
+          // Add cookie consent CSS style
+          $assets->addCss('//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.3/cookieconsent.min.css');
+        }
+
+        // Load cookie consent JS if enabled
+        if (!$this->isAdmin() && $builtInJs) {
```

Comment:
Looks like this if-condition is identical to the one in line 70.

Reply:
idem